test/DomainRegistryV2.test.ts

```diff
@@ -0,0 +1,307 @@
+import {loadFixture,} from "@nomicfoundation/hardhat-toolbox/network-helpers";
```

Comment:
// test
// filed test case
// 1) Should withdraw reward balance for a domain holder, only by owner
// 19 passing (1s)
// 1 failing

// // - 5 points

Reply:
done